artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/mirror/AMQPMirrorControllerTarget.java

```diff
@@ -120,6 +120,11 @@ ACKMessageOperation setDelivery(Delivery delivery) {
 
       @Override
       public void run() {
+         if (!connection.isHandler()) {
+            logger.info("Moving execution to proton handler");
```

Comment:
Should this be info? Feels more like its either expected to happen, and so e.g trace or debug, or not expected and so perhaps even warn.

Reply:
@gemmellr thanks.. it was meant to be a logger.debug... I will send a commit fixing it.